tagr/tagging/artifacts.py

```diff
@@ -131,9 +133,9 @@ def flush(self, proj, exp, dump='local', tag=None):
             # Push dfs #
             #############
             # todo: save larger dfs as parquet, maybe partition as well
-            logger.info("saving dataframes as csv to " + str(dump))
+            logger.info("saving/flushing dataframes as csv to " + str(dump))
```

Comment:
nit: this is confusing. choose save or flush. on line 157 we only have "flushing"